hbw/app/javascript/packs/hbw/components/button.js.jsx

```diff
@@ -1,9 +1,8 @@
 import { withCallbacks } from './helpers';
+import { Component } from 'react';
```

Comment:
Place external import before internal if you don't mind.

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#importorder-enforce-a-convention-in-module-import-order